Cannot find module 'rrweb-snapshot' or its corresponding type declarations

Hi,

I get the following error when trying to upgrade to a later version than `v1.9.7`

```
Error: node_modules/posthog-js/dist/module.d.ts:4:50 - error TS2307: Cannot find module 'rrweb-snapshot' or its corresponding type declarations.

4 import { MaskInputOptions, SlimDOMOptions } from 'rrweb-snapshot'
```

this is for an Angular 12 project

It looks related to this:

https://github.com/PostHog/posthog-js/blob/923e82d498eb6a80d4f185c368b120ff9587ee95/src/module.d.ts#L4

Should [rrweb-snapshot](https://github.com/rrweb-io/rrweb-snapshot) be added to the dependencies?

FYI it works fine if I install `rrweb-snapshot` separately.

--- a/package.json
+++ b/package.json
@@ -89,6 +89,7 @@
         "clearMocks": true
     },
     "dependencies": {
-        "fflate": "^0.4.1"
+        "fflate": "^0.4.1",
+        "rrweb-snapshot": "^1.1.7"
     }
 }

--- a/package.json
+++ b/package.json
@@ -56,6 +56,7 @@
         "prettier": "^2.0.5",
         "rollup": "^2.18.2",
         "rrweb": "^0.9.9",
+        "rrweb-snapshot": "^1.1.7",
         "sinon": "9.0.2",
         "testcafe": "^1.10.1",
         "testcafe-browser-provider-browserstack": "^1.13.2-alpha.1",
@@ -89,7 +90,6 @@
         "clearMocks": true
     },
     "dependencies": {
-        "fflate": "^0.4.1",
-        "rrweb-snapshot": "^1.1.7"
+        "fflate": "^0.4.1"
     }
 }

